src/lib/modules/Patron/PatronBulkExtendLoans/PatronBulkExtendLoans.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon, List, Modal } from 'semantic-ui-react';
+
+export default class PatronBulkExtendLoans extends Component {
+  state = { open: false };
+
+  open = () => this.setState({ open: true });
+  close = () => this.setState({ open: false });
+
+  handleSubmitExtend = () => {
+    const { bulkLoanExtension, patronPid } = this.props;
+
+    bulkLoanExtension(patronPid);
+    this.close();
+  };
+
+  render() {
+    const { patronPid, bulkLoanExtension, isLoading, ...props } = this.props;
+    const { open } = this.state;
+    return (
+      <Modal
+        open={open}
+        onClose={this.close}
+        onOpen={this.open}
+        trigger={
+          <Button
+            labelPosition="left"
+            fluid
+            icon
+            primary
+            loading={isLoading}
+            {...props}
+          >
+            <Icon name="refresh" />
+            Extend all loans
+          </Button>
+        }
+      >
+        <Modal.Header>Confirm extension action</Modal.Header>
+        <Modal.Content>
+          <Modal.Description>
+            The action will extend only the loans:{' '}
+            <List bulleted>
+              <List.Item>
+                <b>with the upcoming date of return,</b>
+              </List.Item>
+              <List.Item>
+                the loans on the physical items <strong>not</strong> obtained
+                from a third party library,
+              </List.Item>
+              <List.Item>
+                and if the <b>book is not in the high demand.</b>
+              </List.Item>
+            </List>
+            Do you want to continue?
+          </Modal.Description>
+        </Modal.Content>
+        <Modal.Actions>
+          <Button onClick={this.close}>Cancel</Button>
+          <Button onClick={this.handleSubmitExtend} primary>
+            Extend the loans
```

Comment:
Does it "sound" better without the `the`?
```suggestion
            Extend loans
```